changelog.d/11429.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in 1.47.0 where `send_join` could fail due to an outdated `ijson` version.
```

Comment:
CI's failing because the newsfile ought to be called `11439.bugfix` (`3` instead of `2`)

Reply:
Fixed, thanks for noticing!